### ISSUE TITLE
Passing a tuple as index to TrajectoryIterator causes memory crash 

### ISSUE THREAD

**Issue by pindakaas42** [Amber-MD/pytraj#1511]
Just try with some trajectory iterator traj.

to do:
`traj[(1,)]`

This causes a crash.

This is in version pytraj-2.0.5-py2.7-linux-x86_64.egg from amber

**Comment by hainm** [comment]
thank you for your report.

**Comment by hainm** [comment]
fixed in #1538 

```
In [4]: traj[(1,)]                                                                                                                                            
Out[4]: <Frame with 223 atoms>
```

Thanks @pindakaas42 for your report.
